docs/01-app/01-getting-started/03-layouts-and-pages.mdx

```diff
@@ -324,3 +324,38 @@ export default async function Post({ post }) {
 ```
 
 > **Good to know**: `<Link>` is the primary way to navigate between routes in Next.js. You can also use the [`useRouter` hook](/docs/app/api-reference/functions/use-router) for more advanced navigation.
+
+## Route Props Helpers
+
+Next.js exposes utility types that infer `params`, `searchParams`, and named slots from your route structure:
```

Comment:
```suggestion
Next.js exposes utility types that infer `params` and named slots from your route structure:
```
Maybe we don't mention searchParams here since they are still generically typed and not expanded on below